.github/workflows/jules-ci-feedback.yml

```
@@ -0,0 +1,57 @@
+          if [ -z "$PR_NUMBER" ]; then
+            echo "No open PR found for branch $HEAD_BRANCH. Skipping notification."
+            exit 0
+          fi
+          
+          echo "Found PR #$PR_NUMBER for branch $HEAD_BRANCH"
+          
+          # Extract failed logs (last 40 lines of failures)
+          FAILED_LOG=$(gh run view "$RUN_ID" --log-failed 2>&1 | tail -n 40 || echo "Logs unavailable")
+          
+          COMMENT_BODY=$(cat <<EOC
+🤖 **CI Auto-Fixer Alert — Checks Failed**
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)